Update README with new akkasls and curl command (#19)
lightbend-labs/akkaserverless-wirelessmesh-java: README.md

### Testing your service

To test using Postman(or curl).
* First install Postman, [found here](https://www.postman.com)
* Assuming you have deployed to akkaserverless and exposed your service to 'winter-mountain-2372.us-east1.apps.akkaserverless.com'...
* Create a Postman POST request to 'https://winter-mountain-2372.us-east1.apps.akkaserverless.com/wirelessmesh/add-customer-location' with the json body '{"customer_location_id": "my-first-location", "access_token": "my lifx access token if applicable in alphanumeric"}'

> Or use curl command (NOTE: Assume access_token is "abcd1234", you can set it to any alphanumeric)

Reviewer: ```suggestion
Or using Curl (NOTE: Assume access_token is "abcd1234", you can set it to any alphanumeric):
```

```
export AS_HOST=winter-mountain-2372.us-east1.apps.akkaserverless.com

curl -X POST -H "Content-Type: application/json"  --data '{"customer_location_id": "my-first-location", "access_token": "abcd1234"}' https://${AS_HOST}/wirelessmesh/add-customer-location
```
* You should see a response of '200(OK) {}', this will be the response of any POST
* You can now create a GET request to 'https://winter-mountain-2372.us-east1.apps.akkaserverless.com/wirelessmesh/get-customer-location?customer_location_id=my-first-location'

> Or use curl command

Reviewer: ```suggestion
Or using Curl:
```

```
curl https://${AS_HOST}/wirelessmesh/get-customer-location?customer_location_id=my-first-location
```
* You should see a json response containing your customer location and no devices.
* Create a POST request to 'https://winter-mountain-2372.us-east1.apps.akkaserverless.com/wirelessmesh/activate-device' with the body '{"customer_location_id": "my-first-location", "device_id": "my-first-device"}'

> Or use curl command

Reviewer: ```suggestion
Or using Curl:
```

```
curl -X POST -H "Content-Type: application/json"  --data '{"customer_location_id": "my-first-location", "device_id": "my-first-device"}' https://${AS_HOST}/wirelessmesh/activate-device
```
* Create a POST request to 'https://winter-mountain-2372.us-east1.apps.akkaserverless.com/wirelessmesh/assign-room' with the body '{"customer_location_id": "my-first-location", "device_id": "my-first-device", "room": "office"}'

> Or use curl command

Reviewer: ```suggestion
Or using Curl:
```

```
curl -X POST -H "Content-Type: application/json"  --data '{"customer_location_id": "my-first-location", "device_id": "my-first-device", "room": "office"}' https://${AS_HOST}/wirelessmesh/assign-room
```
* Create a POST requset to 'https://winter-mountain-2372.us-east1.apps.akkaserverless.com/wirelessmesh/toggle-nightlight' with the body '{"customer_location_id": "my-first-location", "device_id": "my-first-device"}'

> Or use curl command

Reviewer: ```suggestion
Or using Curl:
```

```
curl -X POST -H "Content-Type: application/json"  --data '{"customer_location_id": "my-first-location", "device_id": "my-first-device"}' https://${AS_HOST}/wirelessmesh/toggle-nightlight
```
* Rerun your get-customer-location request

> For example,

Reviewer: ```suggestion
Or using Curl:
```

```
curl https://${AS_HOST}/wirelessmesh/get-customer-location?customer_location_id=my-first-location
```
* You should see a json response with your customer location and a collection of your single device with the room assigned and the nightlight on
* Create a POST request to 'https://winter-mountain-2372.us-east1.apps.akkaserverless.com/wirelessmesh/remove-device' with the body '{"customer_location_id": "my-first-location", "device_id": "my-first-device"}'

> For example,

Reviewer: ```suggestion
Or using Curl:
```

```
curl -X POST -H "Content-Type: application/json"  --data '{"customer_location_id": "my-first-location", "device_id": "my-first-device"}' https://${AS_HOST}/wirelessmesh/remove-device
```
* You should see a json response no longer containing any devices
* Create a POST request to 'https://winter-mountain-2372.us-east1.apps.akkaserverless.com/wirelessmesh/remove-customer-location' with the body '{"customer_location_id": "my-first-location"}'

> For example,

Reviewer: ```suggestion
Or using Curl:
```

```
curl -X POST -H "Content-Type: application/json"  --data '{"customer_location_id": "my-first-location"}' https://${AS_HOST}/wirelessmesh/remove-customer-location
```
* Rerun your get-customer-location request and you will see a server error since it no longer exists.

> For example,

Reviewer: ```suggestion
Or using Curl:
```
